gradio/utils.py

```diff
@@ -958,6 +960,62 @@ def is_in_or_equal(path_1: str | Path, path_2: str | Path):
     return True
 
 
+@document()
+def set_static_paths(paths: list[str | Path]) -> None:
+    """
+    Set the static paths to be served by the gradio app.
```

Comment:
From what I understand (and testing the code), this method sets the static_paths for all Gradio apps defined until that Python session ends (or `set_static_paths` is called again(. For example, if you have multiple Gradio apps in one jupyter notebook, this will affect all of them. We should explicitly describe this behavior here imo.

Reply:
Yea good call!